jobsdb/jobsdb_backup_test.go

```diff
@@ -163,7 +163,7 @@ func TestBackupTable(t *testing.T) {
 	goldenStatusFile, err := tc.readGzipStatusFile(goldenFileStatusFileName)
 	require.NoError(t, err, "expected no error while reading golden status file")
 	verifyStatus(t, backedupStatus, goldenStatusFile)
-	require.Equal(t, len(goldenStatusFile)*2, len(backedupStatus), "expected status files to be same")
+	require.Equal(t, len(goldenStatusFile), len(backedupStatus), "expected status files to be same")
```

Comment:
Don't we need a test for `clean up done before back up` scenario?

Reply:
This test verifies the same.
The only change in behaviour is that two statuses per job would get uploaded earlier, whereas now only one is. These changes reflect that.